hack/unwanted-dependencies.json

```diff
@@ -1,5 +1,15 @@
 {
   "spec": {
+    "pinnedModules": {
+      "github.com/ishidawataru/sctp": {
+        "Version": "v0.0.0-20230406120618-7ff4192f6ff2",
+        "Reason": "broken module, do not update until https://github.com/ishidawataru/sctp/pull/75 is fixed"
+      },
+      "github.com/libopenstorage/openstorage": {
+        "Version": "v1.0.0",
+        "Reason": "portworx csi driver has been deprecated and will be removed soon"
```

Comment:
is there a tracking issue we can link to? "soon" will not age well :)

Reply:
looks like closest would be https://github.com/kubernetes/enhancements/issues/2589

Reply:
done